### DISCUSSION TITLE
How do I keep Skype from running at startup?

### DISCUSSION THREAD

**Discussion by Konglomneshued** [i3/i3#5565] [Q&A] [answered]
I'm deeply confused how Skype is able to launch on its own when there is not a way to manage startup apps in i3.

**Comment by okraits** [comment, marked as answer]
Could you please elaborate a bit more? Maybe it is run by XDG Autostart: https://wiki.archlinux.org/title/XDG_Autostart

**Reply by Konglomneshued** [reply]
Thank you, it was.
